jekyll/_includes/snippets/runner/launch-agent-download.adoc

```diff
@@ -89,16 +89,19 @@ endif::[]
 | 1.0.29477-605777e
 
 | 3.4
-| 1.1.63254-c1aa872
+| 1.1.73695-40bf772
 
 | 4.0
-| 1.1.63254-c1aa872
+| 1.1.73695-40bf772
 
 | 4.1
-| 1.1.63254-c1aa872
+| 1.1.73695-40bf772
 
 | 4.2
-| 1.1.63254-c1aa872
+| 1.1.73695-40bf772
+
+| 4.3
+| 1.1.73695-40bf772
```

Comment:
I think 4.3 gets a newer version than this

Reply:
Good catch. That actually was a copy-paste error on my part, as it seems we forgot to add entries for 4.3. We should pin to [1.1.73189-8792751](https://circleci.slack.com/archives/C03F8RBUMM1/p1702558123548619), although `latest` should also just work with server 4.3 as we start to EoL launch-agent.